lib/WasmReader/WasmByteCodeGenerator.cpp

```diff
@@ -1591,6 +1596,8 @@ EmitInfo WasmBytecodeGenerator::EmitV8X16Shuffle()
     EmitInfo arg2Info = PopEvalStack(WasmTypes::M128);
     EmitInfo arg1Info = PopEvalStack(WasmTypes::M128);
 
+    // FIXME Release arg2Info and arg1Info
```

Comment:
FYI @MikeHolman, releasing those currently triggers seemingly unrelated issues with liveness analysis, I am looking into it